app/components/Button/index.jsx

```diff
@@ -0,0 +1,38 @@
+import cx from 'classnames'
+import React from 'react'
```

Comment:
Yeah, I did that in some of zanata-ui, but most of the time there usually isn't enough different PropTypes to make it worth doing and then you have 2 different ways of doing it. I've found just using PropTyes has been the best balance for me.